src/test/core/analyzer_test.ts

```diff
@@ -257,6 +257,28 @@ suite('Analyzer', () => {
           assert.equal(subBehavior!.className, 'TestBehavior');
         });
 
+    test(
+        'an inline feature can narrow down its containing document',
```

Comment:
Recommend using a `testName` local to store the test name so that indentation doesn't get wonky.